### ISSUE TITLE
Writing a book and quill in 1.15 makes you crash your entire game.

### ISSUE THREAD

**Issue by CrazyMoneyy** [PirateCraft/dev#25]
**Describe the bug** *(A clear and concise description of what the bug is)*:
Trying to write a book in 1.15 is impossible, in odd ways it makes you crash your entire game. Tried it 6 times myself and crashed 6 times within a few seconds from opening the book and writing something. 

It seems that especially using backspace to remove something you had already written makes you crash instantly. 
![image](https://user-images.githubusercontent.com/69697407/90296434-b95e3200-de8b-11ea-9dba-eb61ac844073.png)

**To Reproduce** *(Steps to reproduce the behavior)*:
Startup minecraft 1.15 and get yourself a book and quill. Open the book and write something down. You notice that this is possible write further. Once you have typed a few things especially in the bottom part of the book also try to remove some things using backspace as if you have typed something wrong. Removing things made me crash instantly.

**Time and Date** *(If applicable, add the time (in GMT+1) and date (DD.MM.YYYY) of the issue)*:
23.00 14 August 2020
**Screenshots/Videos (Delete if not needed)** *(If applicable, add screenshots/videos to help explain your problem)*:
Video of what happens + crashcode [ezyzip.zip](https://github.com/PirateCraft/dev/files/5077641/ezyzip.zip)

**Additional context (Delete if this isn't needed)** *(Add any other context about the problem here)*:
The top part of the book is not the problem but when you try to remove some text in the bottom part of the book it crash instantly. Typing in the top part of the book also crashed me once but you were still able to remove some things.

**Comment by FrozenBeard** [comment]
This could be anything. 
I would suspect its ViaVersion, what version exactly, 1.15.2?

**Comment by CrazyMoneyy** [comment]
Yes I know it was very odd and strange how it makes you crash. I tried to e as specific and clear as possible. I was using exactly 1.15 when testing.
